pkg/kfapp/aws/eks.go

```diff
@@ -47,12 +49,26 @@ func (aws *Aws) getEksCluster(clusterName string) (*Cluster, error) {
 
 // IsEksCluster checks if an AWS cluster is EKS cluster.
 func (aws *Aws) IsEksCluster(clusterName string) (bool, error) {
+	session := session.Must(session.NewSession())
+
+	svc := sts.New(session)
+	stsinput := &sts.GetCallerIdentityInput{}
+	result, err := svc.GetCallerIdentity(stsinput)
+
+	if err != nil {
+		log.Infoln("AWS Credentials seems not correct %v", err.Error())
```

Comment:
If we found that AWS Cred seems not correct with the error, should we then `return false, err`?  /cc @Jeffwan

Reply:
Hey, this is not the way we like to move forward. You'd better read region from config and create regional session and used by all AWS service client. 

Or use provider chain to find env, profile, ec2 metadata service. etc.

Reply:
Make sense. Propose new version to improve this.